### PR TITLE
Integration Tests

### PR DESCRIPTION
**Pull request by amore17**
1: Added test with UID and GID for owner and group.
2: Added test to check default auto-renew is set or not.

### CODE REVIEW

**Comment by pcahyna** [comment]
[citest commit:6cfcd003514ad65b2b7e762f4dc7903f891d2521]

**Comment by pcahyna** [comment]
@amore17 @seocam are those test failures expected?

File: tests/tasks/presetup.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (seocam) [inline tests/tasks/presetup.yml:1]: Is this something we are going to reuse in multiple playbooks?
If we are only going to use on `tests/tests_fs_attrs.yml` I would just define it there.

File: tests/tests_fs_attrs.yml [hunk 2]
```diff
@@ -9,6 +16,12 @@
         owner: ftp
         group: ftp
         ca: self-sign
+      - name: certid
```
Comment (seocam) [inline tests/tests_fs_attrs.yml:19]: You need to verify this certificate in the task "Verify Certificate"

File: tests/tests_no_auto_renew.yml [hunk 3]
```diff
@@ -24,6 +27,7 @@
         subject_alt_name:
           - DNS:www.example.com
         auto_renew: no
+      - path: /tmp/defaultcert
```
Comment (seocam) [inline tests/tests_no_auto_renew.yml:30]: You need to add the `.crt` in the path. It would be also good to verify key-path and if `auto_renew: yes`

**Comment by seocam** [comment]
> @amore17 @seocam are those test failures expected?

All Fedoras and CentOS/RHEL 8 should be passing.

**Pull request merged**